app/code/Codifi/Sales/Setup/Patch/Data/AddOrderTypeToCoreConfigData.php

```diff
@@ -43,11 +43,11 @@ class AddOrderTypeToCoreConfigData implements DataPatchInterface
     private $orderTypeValue = [
         [
             'value' => 'REGULAR',
-            'description' => 'Regular Order'
+            'label' => 'Regular Order'
```

Comment:
If data patch is already applied, you can't made updates in it. If you want to update data in db, you need to write new data patch and set appropriate dependency.